tests/test.sh

```diff
@@ -287,71 +93,77 @@ pkeyutl -verify -inkey "${PUBURI}"
                 -rawin
                 -sigfile ${TMPPDIR}/sha256-dgstsig.bin'
 
-title PARA "DigestSign and DigestVerify with RSA PSS"
-ossl '
-pkeyutl -sign -inkey "${BASEURI}"
-              -digest sha256
-              -pkeyopt pad-mode:pss
-              -pkeyopt mgf1-digest:sha256
-              -pkeyopt saltlen:digest
-              -in ${TMPPDIR}/64krandom.bin
-              -rawin
-              -out ${TMPPDIR}/sha256-dgstsig.bin'
-ossl '
-pkeyutl -verify -inkey "${BASEURI}" -pubin
-                -digest sha256
-                -pkeyopt pad-mode:pss
-                -pkeyopt mgf1-digest:sha256
-                -pkeyopt saltlen:digest
-                -in ${TMPPDIR}/64krandom.bin
-                -rawin
-                -sigfile ${TMPPDIR}/sha256-dgstsig.bin'
-title LINE "Re-verify using OpenSSL default provider"
-#(-pubin causes us to export a public key and OpenSSL to import it in the default provider)
-ossl '
-pkeyutl -verify -inkey "${PUBURI}"
-                -pubin
-                -digest sha256
-                -pkeyopt pad-mode:pss
-                -pkeyopt mgf1-digest:sha256
-                -pkeyopt saltlen:digest
-                -in ${TMPPDIR}/64krandom.bin
-                -rawin
-                -sigfile ${TMPPDIR}/sha256-dgstsig.bin'
+if [ "$NOPSS" != "1" ]; then
```

Comment:
From previous comment on test controlling variables this would become:
if [ "$TEST_RSAPSS" != "0" ]; then

Which I find a lot more readable and understandable then a "double negation".